iaac/terraform/aws-infra/rds/main.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_db_subnet_group" "rds_db_subnet_group" {
```

Comment:
can you confirm that this configuration matching the one from auto-rds-s3 script?

Reply:
https://github.com/awslabs/kubeflow-manifests/blob/main/tests/e2e/utils/rds-s3/auto-rds-s3-setup.py#L218-L229

Reply:
https://github.com/awslabs/kubeflow-manifests/blob/main/tests/e2e/utils/rds-s3/auto-rds-s3-setup.py#L218-L229

Reply:
added the following options from the script
```
BackupRetentionPeriod=DB_BACKUP_RETENTION_PERIOD,
StorageType=DB_STORAGE_TYPE,
DeletionProtection=True,
MaxAllocatedStorage=DB_MAX_STORAGE,
PubliclyAccessible=False,
```